compatibility_lib/compatibility_lib/mock_data.py

```diff
@@ -0,0 +1,470 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""mock data for testing"""
```

Comment:
Can we fake some data for unstable release? We don't need to use the actual data for test, just need to make sure we cover all the conditions in the check.

Reply:
I have dummy data for that test here: https://github.com/ylil93/cloud-opensource-python/blob/3bf8977e25bde38381ae20cbd6a1462c79611185/compatibility_lib/compatibility_lib/test_dependency_highlighter.py#L286